src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -850,24 +883,29 @@ export function AppNotConfigured({
     subtitle,
     buttonTitle,
     appConfigTabs ='',
+    style
 }: {
     image?: any
     title?: string
     subtitle?: string
     buttonTitle?: string
     appConfigTabs?: string
+    style?:{}
 }) {
     const { appId } = useParams<{ appId: string }>()
     const { push } = useHistory()
     function handleEditApp(e) {
         getAppConfigStatus(+appId).then((response) => {
-            let url = `/app/${appId}/edit${appConfigTabs}`
+            let url = `/app/${appId}/edit`
+            if(appConfigTabs){
+                url = `/app/${appId}/${appConfigTabs}`
```

Comment:
Earlier  url = `/app/${appId}/edit${appConfigTabs}` , As per recent changes you have removed `edit` from URL . Is this known?

Reply:
We are redirecting user to deploy & build tab. Hence removing edit.